crates/core/src/host/module_host.rs

```diff
@@ -725,6 +671,26 @@ pub struct CallProcedureParams {
     pub args: ArgsTuple,
 }
 
+impl CallProcedureParams {
+    /// Returns a set of parameters for a call that came from within
+    /// and without a client/caller/request_id.
```

Comment:
```suggestion
    /// Returns a set of parameters for an internal call without a client/caller/request_id.
```
"within and without" is a little too poetic for a doc comment, I think.